src/olympia/devhub/tests/test_utils.py

```diff
@@ -98,14 +102,16 @@ def check_file(self, file_):
 
 
 class TestValidatorListed(TestValidatorBase):
-    @mock.patch('olympia.devhub.utils.chain')
-    def test_run_once_per_file(self, chain):
+    @mock.patch('celery.Signature.__or__')
+    def test_run_once_per_file(self, chain_mock):
```

Comment:
This seems to make this whole thing much more complex and either needs to be rethought or documented why we're mocking a celery internal function here. I know that this is hooking into the `|` (chain-operator in celery-task speak) but that's definitely not obvious.

Reply:
we were mocking the internal function already ('olympia.devhub.utils.chain').  I could use `chain` rather than `|` directly if you want.